### DISCUSSION TITLE
Abstract for MSWW

### DISCUSSION THREAD

**Discussion by FranckCo** [linked-statistics/COOS#82] [General]
The ModernStats standard models have been developed over the years in a mostly independent way, and with little reference to external initiatives. More recently, an effort was put into coordination of the models in order to provide a more integrated view.

The development of the Core Ontology for Official Statistics (COOS) is an element of this effort. A dedicated Task Team, launched in 2021 under the Supporting Standards Group, has delivered first versions of key outputs: an ontology specification, a governance document, a URI policy and an OWL ontology. The COOS activity continues in 2022 with specific objectives of integrating feedback from a review by external experts, of developing concrete use cases and improving the communication and promotion of the ontology.

The presentation will give an overview of COOS and its positioning in the ModernStats standardization activities, and show by examples how it can be leveraged in different use cases.

**Comment by JALinnerud** [comment]


**Reply by FranckCo** [reply]
Hi @JALinnerud 
Could you move that to #76?

**Comment by FlavioRizzolo** [comment]
Instead of "an effort was put into coordination of the models", how about "an effort has been put towards aligning the models"?

**Reply by FranckCo** [reply]
Still making these rookie mistakes after all these years... :-(
